src/DefaultBuilder/src/ContentRootHelpers.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore
+{
+    internal class ContentRootHelpers
```

Comment:
ContentRootResolver or something like that? Helpers is a bad name. Also static

Reply:
Or Utils 😉 . Thanks @rynowak